modules/programs/antidote.nix

```
@@ -0,0 +1,71 @@
+{ config, lib, pkgs, ... }:
+
+with builtins;
+with lib;
+let
+  module_config = config.programs.zsh.antidote;
+
+  zPluginStr = with lib;
+    (pluginNames:
+      optionalString (pluginNames != [ ]) "${concatStrings (map (name: ''
+        ${name}
+      '') pluginNames)}");
+  trimLines = (text:
+    concatLines (map (p: removePrefix "     " p) (splitString "\n" text)));
+in {
+  options.programs.zsh.antidote = {
+    enable = mkEnableOption "antidote - a zsh plugin manager";
+
+    plugins = mkOption {
+      default = [ ];
+      type = types.listOf types.str;
+      description = "List of antidote plugins, strings.";
```

No need to say that they are string, the option type already makes that clear.

```suggestion
      description = "List of antidote plugin.";
```